cps/src/main/js/workflow-editor.js

```diff
@@ -48,7 +48,34 @@ jenkinsJSModules.import('ace-editor:ace-editor-122')
                 });
 
                 editor.on('blur', function() {
-                    textarea.trigger('change');
```

Comment:
Well, it's true but `doCheckScript` is still called when you check/uncheck the sandbox checkbox (sorry for the tongue-twister).

Sandbox is activated by default, so if you write a script requiring approval and uncheck the sandbox then regular validation is shown.

Do we want to run sandboxing related validation on textarea changes too? If so, we have to go back to the previous commit and manage both approval and syntax validations in the JSON callback.